implants/lib/eldritch/src/sys.rs

```diff
@@ -59,36 +61,86 @@ impl<'v> UnpackValue<'v> for SysLibrary {
 // This is where all of the "sys.X" impl methods are bound
 #[starlark_module]
 fn methods(builder: &mut MethodsBuilder) {
-    fn exec<'v>(this: SysLibrary, starlark_heap: &'v Heap, path: String, args: Vec<String>, disown: Option<bool>) -> anyhow::Result<Dict<'v>> {
```

Comment:
Lets leave the function definitions on one line too or make all of them multi-line for consistency.

Reply:
https://github.com/rust-lang/rustfmt/blob/master/Configurations.md#vertical